tritium-processor/src/test/java/com/palantir/tritium/processor/TritiumProcessorTest.java

```diff
@@ -113,8 +113,7 @@ public void testDeprecatedInterface() {
 
     @Test
     public void testEmptyInterface() {
-        Compilation compilation = compileTestClass(TEST_CLASSES_BASE_DIR, Empty.class);
-        assertThat(compilation).hadErrorContaining("The annotated interface has no methods");
+        assertTestFileCompileAndMatches(TEST_CLASSES_BASE_DIR, Empty.class);
```

Comment:
While unusual, there's no reason why we can't support empty interfaces. It was a trivial fix, so I've fixed it here.